### DISCUSSION TITLE
How to add css only for Menu manager?

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#4648] [Q&A Core] [answered]
The menu area is hidden because of theme css, but it is there

![image](https://user-images.githubusercontent.com/5429548/148053841-96ba4448-51cc-4c7a-a9d6-0f4250d93baf.png)

after changing some CSS

![image](https://user-images.githubusercontent.com/5429548/148054131-47cfeea2-7892-479d-af45-008e21e4bfb4.png)

But I want to add/change this css only to fix menu manager, not frontend theme. 

If there is no way now, it is possible to add some class to HTML or BODY tag in the menu manager iframe?  

Thanks

**Comment by Moc** [comment]
Could you help me understand why you want want to hide menu areas using CSS rather than using the visibility functionality in the menu manager?

**Reply by Jimmi08** [reply]
Not at all.   Sorry, I probably worded this wrong way.

I have theme and the preview of this theme is breaking menu manager (menu area 21 is not displayed because frontend theme CSS ) - it is not because z-index. 

I know how to fix it with CSS but there is no class or ID in menu manager to apply this fix only for menu manager. 
The last unique id is for IFRAME tag.  But if HTML or BODY tags of the menu manager have a special class, then it can be easily fixed with theme style.css

Example:
style.css
```
.header {
 position: absolute;
}
```

```
.menumanagerclass .header {
 position: initial;
}
```

I am missing that specific class for menu manager.   

But maybe there is some other way, some special CSS or something... 

Or some constant I can use in e_theme class (in theme.php) to load only css fix.

**Comment by Jimmi08** [comment, marked as answer]
This is a working solution:
in theme.php
```

class theme implements e_theme_render
	{

        public function init()
        {

            if(e_PAGE == "menus.php")  {
                    e107::css('theme',  'menumanagerfix.css');  
            }

        }
}
```
